docker/README.md

```diff
@@ -0,0 +1,144 @@
+
+```yaml
+services:
+  dgen:
+    environment:
+      DGEN_DISABLE_AUTO_START: 1
+```
+
+### Customizing the dataset
+
+By default, the dgen container uses the [Deleware residential dataset](https://oedi-data-lake.s3.amazonaws.com/dgen/de_final_db/dgen_db.sql).
+
+You can find more datasets using the links below:
+- [DGen Dataset Submissions on OpenEI](https://data.openei.org/submissions/1931)
+- [DGen Dataset S3 Viewer](https://data.openei.org/s3_viewer?bucket=oedi-data-lake&prefix=dgen%2F)
+
+You can customize the dataset used by overriding the DGEN_DATAFILE_URL and DGEN_AGENTFILE_URL variables in `docker-compose.yml` and then editing `~/dgen_data/input_sheet_final.xlsm` using Excel.
+
+Below will walk through the process of using the [Colorado residential dataset](https://oedi-data-lake.s3.amazonaws.com/dgen/co_final_db/dgen_db.sql).
+
+Update the `docker-compose.yml` to use the co_final_db sql download and set the variable to force remove the database.  This will result in dataloss from previous runs, if this is a concern, please make backups before proceeding with the below steps.
+
+```yaml
+services:
+  postgis:
+    environment:
+      DGEN_DATAFILE_URL: https://oedi-data-lake.s3.amazonaws.com/dgen/co_final_db/dgen_db.sql
+      DGEN_AGENTFILE_URL: https://oedi-data-lake.s3.amazonaws.com/dgen/co_final_db/agent_df_base_res_co_revised.pkl
+      DGEN_FORCE_DELETE_DATABASE: 1 # Clear all the data in the database to reload the Colorado dataset, Warning this will remove your existing data.
+```
+
+Edit the excel document `~/dgen_data/input_sheet_final.xlsm` using Excel (Enable macros), edit the Region to Analyize to `Colorado` and Markets to `Only Residential`, then click Save Scenario.
+
+Restart your containers with the above options.  This will remove all your existing data and download the new Colorado dataset.
+
+```bash
+$ docker-compose down
+[+] Running 3/3
+ ✔ Container dgen_1  Removed  9.2s
+ ✔ Container postgis_1  Removed  0.1s
+
+ $ docker-compose up -d
+ [+] Running 2/2
+ ✔ Container dgen_1     Started 0.1s
+ ✔ Container postgis_1  Started 0.2s
+ ```
+
+After you load the new Colorado dataset, remove the DGEN_FORCE_DELETE_DATABASE option to prevent future accidental data loss.
+
+```yaml
+services:
+  postgis:
+    environment:
+      DGEN_FORCE_DELETE_DATABASE: 0
+```
+
+You can now attach to the dgen container and monitor the data download.  This may take 5-10 minutes depending on your internet speed, if the file size is increasing its still downloading.
+
+```bash
+$ docker attach dgen_1
+(dg3n) dgen@cc6e2e5f70b5:/opt/dgen_os/python$ ls -lh /data/dgen_db.sql
+-rw-r--r-- 1 dgen dgen 705M Jan 29  2025 /data/dgen_db.sql
+(dg3n) dgen@cc6e2e5f70b5:/opt/dgen_os/python$ python dgen_model.py # Run scenario
+(dg3n) dgen@cc6e2e5f70b5:/opt/dgen_os/python$ exit # to exit
+```
+
+### Stop running containers
+```bash
+$ docker ps -a
```

Comment:
For this block, would it be better/possible to separate the commands from the returned statements?

Reply:
I seperated out the code blocks for commands vs output, I do agree that it reads better with the separation. Let me know what you think.